packages/showcases/svelte/stories/components/navigation/VtmnSearch.stories.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
+  import { VtmnSearch } from '@vtmn/svelte';
+</script>
+
+<Meta
+  title="Components / Navigation / VtmnSearch"
+  component={VtmnSearch}
+  argTypes={{
+    variant: {
+      type: { name: 'string', required: true },
+      description: 'Color variant of search component.',
+      control: {
+        type: 'select',
+        options: ['default', 'ghost', 'persistent', 'on-content'],
+      },
+      defaultValue: 'default',
+    },
+    disabled: {
+      type: { name: 'boolean', required: false },
+      description: 'Disabled state',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    size: {
+      type: { name: 'string', required: false },
+      description: 'Size of input.',
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium'],
+      },
+    },
+  }}
+  parameters={{
+    actions: {
+      handles: ['mouseover', 'click', 'focus', 'keypress', 'keydown'],
+    },
+  }}
```

Comment:
Can you please move `argTypes` and `parameters` in the `showcases/core` .csf file as we did with other components in order to centralize this data? Thanks

>